lib/facade/MonitoringAspect.ts

```diff
@@ -320,6 +320,12 @@ export class MonitoringAspect implements IAspect {
         alarmFriendlyName: node.node.path,
         ...props,
       });
+    } else if (isEnabled && node instanceof rds.DatabaseInstance) {
```

Comment:
Similar to the splitting of `rds` to `rdsCluster`/`rdsInstance`, make `monitorRds` into 2 methods too using the updated props.